.travis.coverage.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+if [ "$TRAVIS_PHP_VERSION" = '7.1' ] ; then
```

Comment:
This will run the code for ALL PHP 7.1 builds, it would be better to have it only for specific platform as well.